beacon_node/network/src/network_beacon_processor/rpc_methods.rs

```diff
@@ -893,6 +951,49 @@ impl<T: BeaconChainTypes> NetworkBeaconProcessor<T> {
         );
 
         let request_start_slot = Slot::from(req.start_slot);
+        let request_start_epoch = request_start_slot.epoch(T::EthSpec::slots_per_epoch());
+        // Should not send more than max request blob sidecars
+        if req.max_blobs_requested(request_start_epoch, &self.chain.spec)
+            > self.chain.spec.max_request_blob_sidecars_electra
```

Comment:
should we use the `spec.max_request_blob_sidecars(fork_name)` function here?
keeping those fields private means we don't leak the logic elsewhere -if we ever need to use a different config value for a different fork, we only need to update the above function.